db/l2db/l2db.go

```diff
@@ -125,7 +126,7 @@ func (l2db *L2DB) UpdateTxsInfo(txs []common.PoolL2Tx) error {
 	}
 	txUpdates := make([]txUpdate, len(txs))
 	for i := range txs {
-		txUpdates[i] = txUpdate{ID: txs[i].TxID, Info: txs[i].Info}
+		txUpdates[i] = txUpdate{ID: txs[i].TxID, Info: "BatchNum: " + strconv.FormatInt(int64(batchNum), 10) + ". " + txs[i].Info}
```

Comment:
since `"BatchNum: " + strconv.FormatInt(int64(batchNum), 10) + ". "` will be the same for all the iterations of the loop, it would be better to save this in a var outside the loop